Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/autorefinement.h

```diff
@@ -90,6 +89,11 @@ struct Default_visitor
   inline void number_of_output_triangles(std::size_t /*nbt*/) {}
   inline void verbatim_triangle_copy(std::size_t /*tgt_id*/, std::size_t /*src_id*/) {}
   inline void new_subtriangle(std::size_t /*tgt_id*/, std::size_t /*src_id*/) {}
+  inline void delete_triangle(std::size_t /*src_id*/) {}
+
+  // Not documented
+  template< typename Triangle>
+  inline void internal_new_subtriangle(Triangle& /*new_t*/ , const Triangle& /*old_t*/) {}
```

Comment:
```suggestion
```

Reply:
you should only have it your special visitor, and use constexpr if to only call it when you can